interp/src/macros.rs

```diff
@@ -192,3 +225,10 @@ macro_rules! validate {
         }
     }
 }
+
+#[macro_export]
```

Comment:
Document macro and explain what it is being used for. Also, this seems simple enough to not be a macro?

Reply:
whoops forgot to delete this, thanks for the reminder